src/main/java/com/virtuslab/using_directives/custom/Parser.java

```diff
@@ -170,10 +170,13 @@ UsingPrimitive primitive(int keyEnd) {
     int offset = offset(in.td.offset);
     UsingPrimitive res = null;
     if (in.td.token == Tokens.STRINGLIT) {
-      res = new StringLiteral(in.td.strVal, source.getPositionFromOffset(offset));
+      res = new StringLiteral(in.td.strVal, source.getPositionFromOffset(offset), false);
+      in.nextToken();
+    } else if (in.td.token == Tokens.STRINGLITDOUBLEQUOTES) {
```

Comment:
Actually, don't we get `IDENTIFIER` in case we are missing double quotes? Maybe it's not needed to add an additional token type?

Reply:
Thanks, good point! I have simplified these changes